tests/functional/basic/Report.lua

```diff
@@ -29,23 +29,33 @@ return {
 		unitTest:assertNil(report.title)
 		unitTest:assertEquals(getn(report.text), 0)
 		unitTest:assertEquals(getn(report.image), 0)
+		unitTest:assertEquals(getn(report.matrix), 0)
+		unitTest:assertEquals(getn(report.graphic), 0)
+		unitTest:assertEquals(getn(report.mult), 0)
 
 		local image = packageInfo("publish").path.."images/urbis_2010_real.PNG"
 		report = Report{title = "My Report"}
 
 		report:addHeading("My Heading")
 		report:addImage(image)
 		report:addText("Some text")
+		report:addMatrix("My table")
```

Comment:
addMatrix does not exist. Please remove this and the other tests that call this function.

Reply:
Okay